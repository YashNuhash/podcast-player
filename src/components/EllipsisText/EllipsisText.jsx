import React from 'react'
import styles from './EllipsisText.module.scss'

export const EllipsisText = ({ tagName = 'span', children, className = '' }) => {
  return React.createElement(tagName, { className: `${styles.Ellipsis} ${className}` }, children)
}

export default EllipsisText
