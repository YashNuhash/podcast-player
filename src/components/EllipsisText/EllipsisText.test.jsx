import React from 'react'
import { render } from '@testing-library/react'
import { EllipsisText } from './EllipsisText'

it('should render EllipsisText component', () => {
    const text = 'Hello'
    const { getByText } = render(<EllipsisText>{text}</EllipsisText>)
    expect(getByText(text)).toMatchSnapshot()
})
