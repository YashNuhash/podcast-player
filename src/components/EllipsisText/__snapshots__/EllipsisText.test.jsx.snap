// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render EllipsisText component 1`] = `
<span
  class="Ellipsis "
>
  Hello
</span>
`;
